### DISCUSSION TITLE
TypeError: undefined is not an object (evaluating 'depBlock.dependencies') from plugin FlagDependencyExportsPlugin.

### DISCUSSION THREAD

**Discussion by saravanan10393** [webpack/webpack#18766] [Q&A]
I have no idea why / when this error popups up. Any hint will be helpful to resolve this issue.

52 | 						modules,
53 | 						(module, callback) => {
54 | 							const exportsInfo = moduleGraph.getExportsInfo(module);
55 | 							// If the module doesn't have an exportsType, it's a module
56 | 							// without declared exports.
57 | 							if (!module.buildMeta || !module.buildMeta.exportsType) {
                                             ^
TypeError: undefined is not an object (evaluating 'depBlock.dependencies')
      at processDependenciesBlock (/Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/webpack/lib/FlagDependencyExportsPlugin.js:57:41)
      at /Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/webpack/lib/FlagDependencyExportsPlugin.js:335:9
      at /Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/neo-async/async.js:2830:7
      at done (/Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/neo-async/async.js:2865:11)
      at /Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/neo-async/async.js:2818:7
      at /Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/webpack/lib/FlagDependencyExportsPlugin.js:100:10
      at /Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/webpack/lib/cache/IdleFileCachePlugin.js:78:9
      at /Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/webpack/lib/Cache.js:94:6
      at file:///Users/saravanan/projects/saravanan/kf-xg-frontend/node_modules/tapable/lib/HookCodeFactory.js:30:10

**Comment by alexander-akait** [comment]
@saravanan10393 Can you create repreducible test repo?
